### DISCUSSION TITLE
Mod for Windows 10/11 default enhanced Clipboard

### DISCUSSION THREAD

**Discussion by hnavia** [ramensoftware/windhawk-mods#87] [General]
Hi, I love this tool. It’s helped me in many ways.

Unfortunately, I’m not a very good developer so my skills are pretty basic so I can’t create a module for it.

Is it possible to create a mod that allows the Windows 11 clipboard to “refresh” the last used item copíed?

I mean, If I have 3 items in the following order (for example):

A
B
C

And I call the Windows 11 clipboard using Windows key + V and select item C, The item C stays in the same third place. This means that, after several more items copied, the item C will go WAY below third spot.

The idea is to be able to select and paste item C and make the list refresh so it looks like:

C
A
B

Basically, bringing the item used/pasted/selected to be at the top of the list because it was the last item pasted or used.

Is it possible to create such a mod?

Thank you in advance and kind regards.

![image](https://user-images.githubusercontent.com/69368602/211911730-998c1bbb-2cb9-47e8-a839-8b73f572e804.png)
